packages/state-transition/src/util/genesis.ts

```diff
@@ -164,6 +165,16 @@ export function applyDeposits(
     processDeposit(fork, state, deposit);
   }
 
+  // Process deposit balance updates
```

Comment:
@nazarhussain this is to fix e2e, I suppose it's helpful for the spec test as well